### DISCUSSION TITLE
Bug report on OrchardCore.Sitemaps

### DISCUSSION THREAD

**Discussion by DotCat1985** [OrchardCMS/OrchardCore#10856] [Show and tell]
Hi,
I enabled and configured the sitemap using the module **OrchardCore.Sitemaps** and **OrchardCore 1.0 RC2 13450**. It seems to work fine with Google Search Console, but the XML Schema instance is **http://wwww.w3.org/2001/XMLSchema-instance** instead of **http://www.w3.org/2001/XMLSchema-instance**: see the snippets below.

[sitemap_index.xml](https://www.tokyodancemusic.com/sitemap_index.xml)
`<?xml version="1.0" encoding="utf-8"?>
<sitemapindex xmlns:xsi="http://wwww.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.sitemaps.org/schemas/sitemap/0.9 http://www.sitemaps.org/schemas/sitemap/0.9/siteindex.xsd" xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
  <sitemap>
    <loc>https://www.tokyodancemusic.com/pages.xml</loc>
    <lastmod>2021-12-09T22:05:42+00:00</lastmod>
  </sitemap>
</sitemapindex>`

[pages.xml](https://www.tokyodancemusic.com/pages.xml)
`<?xml version="1.0" encoding="utf-8"?>
<urlset **xmlns:xsi="http://wwww.w3.org/2001/XMLSchema-instance"** xsi:schemaLocation="http://www.sitemaps.org/schemas/sitemap/0.9 http://www.sitemaps.org/schemas/sitemap/0.9/sitemap.xsd" xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
  <url>
    <loc>https://www.tokyodancemusic.com/</loc>
    <lastmod>2021-12-09T22:05:42+00:00</lastmod>
    <changefreq>monthly</changefreq>
    <priority>0.5</priority>
  </url>
  <url>
    <loc>https://www.tokyodancemusic.com/bio</loc>
    <lastmod>2021-12-09T21:46:51+00:00</lastmod>
    <changefreq>monthly</changefreq>
    <priority>0.5</priority>
  </url>
  <url>
    <loc>https://www.tokyodancemusic.com/music</loc>
    <lastmod>2021-12-09T21:47:16+00:00</lastmod>
    <changefreq>monthly</changefreq>
    <priority>0.5</priority>
  </url>
  <url>
    <loc>https://www.tokyodancemusic.com/contact</loc>
    <lastmod>2021-12-09T21:47:46+00:00</lastmod>
    <changefreq>monthly</changefreq>
    <priority>0.5</priority>
  </url>
  <url>
    <loc>https://www.tokyodancemusic.com/thank-you</loc>
    <lastmod>2021-12-08T21:57:25+00:00</lastmod>
    <changefreq>monthly</changefreq>
    <priority>0.5</priority>
  </url>
  <url>
    <loc>https://www.tokyodancemusic.com/it/home</loc>
    <lastmod>2021-12-09T21:45:30+00:00</lastmod>
    <changefreq>monthly</changefreq>
    <priority>0.5</priority>
  </url>
  <url>
    <loc>https://www.tokyodancemusic.com/it/bio</loc>
    <lastmod>2021-12-09T21:48:38+00:00</lastmod>
    <changefreq>monthly</changefreq>
    <priority>0.5</priority>
  </url>
  <url>
    <loc>https://www.tokyodancemusic.com/it/musica</loc>
    <lastmod>2021-12-09T21:47:23+00:00</lastmod>
    <changefreq>monthly</changefreq>
    <priority>0.5</priority>
  </url>
  <url>
    <loc>https://www.tokyodancemusic.com/it/contattami</loc>
    <lastmod>2021-12-09T21:47:51+00:00</lastmod>
    <changefreq>monthly</changefreq>
    <priority>0.5</priority>
  </url>
  <url>
    <loc>https://www.tokyodancemusic.com/it/grazie</loc>
    <lastmod>2021-12-08T21:57:33+00:00</lastmod>
    <changefreq>monthly</changefreq>
    <priority>0.5</priority>
  </url>
</urlset>`

Is the bug solved in **OrchardCore 1.1.0**? If no, it should be solved because XML Schema instance is linked to wrong URL.
What do you think about it?

**Comment by ns8482e** [comment]
@Tokyo1985 Yes It's fixed #7454
